tests/components/onewire/test_entity_owserver.py

```diff
@@ -194,11 +427,22 @@ async def test_owserver_setup_valid_device(hass, device_id):
 
     assert len(entity_registry.entities) == len(expected_sensors)
 
+    if len(expected_sensors) > 0:
+        device_info = mock_device_sensor["device_info"]
+        assert len(device_registry.devices) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)